File: DotNetUEApp_C#/Models/Enseignant.cs
using System.ComponentModel.DataAnnotations;

namespace DotNetUEApp_C_.Models;

public class Enseignant
{
    public int Id { get; set; }
    public string Lastname { get; set; }
    public string Firstname { get; set; }

    public string Email { get; set; }

    public List<ActiviteApprentissage>? ListActiviteApprentissages { get; set; }

    //public List<UniteEtude>? ListUniteEtudes { get; set; }

}

File: models/enseignant.ts
// enseignant.ts (dans le dossier models)

import { ActiviteApprentissage } from "./activiteApprentissage";

export class Enseignant {
    id!: number;
    lastname!: string;
    firstname!: string;
    email!: string;
    listActiviteApprentissages!: ActiviteApprentissage[] | null;

    // listUniteEtudes: UniteEtude[] | null; // Si vous définissez le modèle pour UniteEtude
  }
  
  
  